.buildkite/scripts/runtask.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cleanup() {
+  echo "trapped exit, cleaning up"
+  task destroy
+  exit $?
+}
+trap cleanup EXIT INT TERM
+
+task demo-standup
+task destroy
```

Comment:
fyi, in case it helps, task has a feature for exposing golang's `defer`. works nicely for this type of scenarios https://taskfile.dev/usage/#doing-task-cleanup-with-defer

Reply:
that can still be a single line.  having a high-level `ci` like:

```yaml
tasks:
  ci:
    desc: run CI tasks
    cmds:
      - defer: { task: destroy }
      - demo-setup
      - test-tls-cluster
```

feel free to ignore though... just giving ideas

Reply:
I think in this case I will unfortunately declare stet 😅. Having stranded resources I couldn't clean up myself was a scarring experience and I don't want to risk repeating it -- even if the alternative is cleaner.